browser/cypress/e2e/home_page.cy.ts

```diff
@@ -0,0 +1,108 @@
+// Sample test for home page
+
+describe('Home Page', () => {
+  // make sure basics of the page succesfully load
+  it('Home page loads successfully', () => {
+    cy.visit('/')
+
+    // Make sure the page has the landing splash
+    cy.contains('Genome Aggregation Database')
+    cy.contains('gnomAD browser')
+
+    // Make sure page content headers are there
+    cy.contains('New to gnomAD?')
+    cy.contains('About')
+  })
+
+  // json for each header page for ease of iterating for tests
+  const headerPages = {
+    pages: [
+      {
+        pageTitle: 'About',
+        pageUrl: '/about',
+        pageHeader: 'About gnomAD',
+      },
+      {
+        pageTitle: 'Team',
+        pageUrl: '/team',
+        pageHeader: 'The gnomAD Team',
+      },
+      // news - in seperate application, test there instead?
+      // changelog - same as above
+      {
+        pageTitle: 'Downloads',
+        pageUrl: '/downloads',
+        pageHeader: 'Google Cloud Public Datasets',
+      },
+      {
+        pageTitle: 'Policies',
+        pageUrl: '/policies',
+        pageHeader: 'gnomAD Ethics Policy',
+      },
+      {
+        pageTitle: 'Publications',
+        pageUrl: '/publications',
+        pageHeader: 'publications by the gnomAD group',
+      },
+      {
+        pageTitle: 'Feedback',
+        pageUrl: '/feedback',
+        pageHeader: 'Tell us how you use gnomAD',
+      },
+      {
+        pageTitle: 'Help',
+        pageUrl: '/help',
+        pageHeader: 'Table of contents',
+      },
+    ],
+  }
+
+  /**
+   * Check that every link on the header correctly navigates to the new page
+   */
+  headerPages.pages.forEach(route => {
+    it(route.pageTitle + ' header link works correctly', () => {
+      cy.visit('/')
+      cy.contains(route.pageTitle).click()
+
+      // ensure the route loads the correct page
+      cy.contains('gnomAD browser')
+      cy.contains(route.pageHeader)
+      cy.url().should('include', route.pageUrl)
+    })
+  })
+
+  // TODO: stub these? so that you see that the page properly loads with dummy data
+  //   However, it'd be good to use real data from the API for well understood
+  //   happy paths so the strategy could be:
+  //   - Stub in general? To not query real API
```

Comment:
Definitely let's stub here, as a rule hitting a remote service in a test suite is setting yourself up for trouble. As a rule anything that makes tests fail intermittently habituates people to test failures, and getting habituated to test failures means missing meaningful test failures.

I can see the argument for going to the trouble for a handful of top-priority happy paths, but I'd also say those are the exact paths that should be tested most thoroughly at the lower levels too.